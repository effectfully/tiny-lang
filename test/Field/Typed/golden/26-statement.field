for x = 1 to 1 do assert T; end; T