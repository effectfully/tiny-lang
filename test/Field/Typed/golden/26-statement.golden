SomeOf Bool (EStatement (EAssert (EConst (UniConst Bool True))) (EConst (UniConst Bool True)))